.github/workflows/ci.yml

```
@@ -0,0 +1,48 @@
+      - name: Run dockerfile_lint
+        run: npx dockerfile_lint
+
+  build-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Get latest jsonlint version
+        id: jsonlint-version
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:2:44: Double quote to prevent globbing and word splitting

(shellcheck)